docs/topics/data-science-overview.md

```diff
@@ -1,70 +1,92 @@
+
+With Kotlin's Exploratory Data Analysis (EDA) tools, such as [Kotlin notebooks](#notebooks), [Kotlin DataFrame](#kotlin-dataframe), and [Kandy](#kandy), you
+have at your disposal a rich set of capabilities to enhance your analytics skills and support you across different scenarios:
+
+* **Load, transform, and visualize data in various formats:** with our Kotlin EDA tools, you can perform tasks like filtering, sorting, and aggregating data. Our tools can seamlessly
+read data right in the IDE from different file formats, including CSV, JSON, and TXT. 
+
+    Kandy, our plotting tool, allows you to create a wide range of charts to visualize and gain insights from the dataset.
+
+* **Efficiently analyze data stored in relational databases:** Kotlin DataFrame seamlessly integrates with databases and provides capabilities similar to SQL queries. 
+You can retrieve, manipulate, and visualize data directly from various databases.
+
+* **Fetch and analyze real-time and dynamic datasets from web APIs:** the EDA tools' flexibility allows integration with external APIs via protocols like OpenAPI. 
+This feature helps you fetch data from web APIs, to then clean and transform the data to your needs.
+
+Would you like to try our Kotlin tools for data analysis?
+
+<a href="get-started-with-kotlin-notebooks.md"><img src="kotlin-notebooks-button.svg" width="600" alt="Get started with Kotlin Notebook"/></a>
+
+Our Kotlin data analysis tools let you smoothly handle your data from start to finish. Effortlessly 
+retrieve your data with simple drag-and-drop functionality in our Kotlin Notebook. Clean, transform, and visualize it with just a few lines of code. 
+Additionally, export your output charts in a matter of clicks.
+
+![Kotlin Notebook](data-analysis-notebook.gif){width=700}
 
 ## Notebooks
 
-_Notebooks_ are interactive editors. Notebooks such as [Kotlin Notebook](https://plugins.jetbrains.com/plugin/16340-kotlin-notebook),
-[Datalore](http://jetbrains.com/datalore)and [Jupyter Notebook](https://jupyter.org/), 
-provide convenient tools for data visualization and exploratory research.
+_Notebooks_ are interactive editors that integrate code, graphics, and text in a single environment. When using a notebook, 
+you can run code cells and immediately see the output. 
 
-Kotlin integrates with these tools to help you explore data, share your findings with 
-colleagues, or build up your data science and machine learning skills.
+Kotlin offers different notebook solutions, such as [Kotlin Notebook](https://plugins.jetbrains.com/plugin/16340-kotlin-notebook), [Datalore](http://jetbrains.com/datalore), 
+and [Jupyter Notebook](https://jupyter.org/), to provide convenient features for data retrieving, transformation, exploration, modeling, and more. 
 
-### Kotlin Notebook
+With our Kotlin notebooks, you can benefit from the features of a powerful interactive editor and the perks of coding with Kotlin. Kotlin integrates with these notebooks 
```

Comment:
As we discussed, let's further discuss if we want to keep the part about the interactive editor here.